src/Cargo.lock

```diff
@@ -1636,8 +1636,8 @@ dependencies = [
 
 [[package]]
 name = "racer"
-version = "2.1.2"
-source = "registry+https://github.com/rust-lang/crates.io-index"
+version = "2.1.3"
+source = "git+https://github.com/racer-rust/racer?rev=2af3ab88420e968500fcfc75418be7e852debfa0#2af3ab88420e968500fcfc75418be7e852debfa0"
```

Comment:
Could we depend on the [released](https://crates.io/crates/racer) racer 2.1.3? Using a git dependency will fail tidy.

Reply:
:-(